bin/localnet.sh

```diff
@@ -76,13 +76,18 @@ up)
   fi
   [[ $channel = edge || $channel = beta ]] || usage "Invalid channel: $channel"
 
+  if [[ -n $2 ]]; then
```

Comment:
I think I'd prefer this to be a named argument rather than positional, to keep the argument namespace open for future expansion.

Something cheesy like this would be enough for now:
```bash
if [[ $2 = -n ]]; then
  [[ -n $3 ]] || usage "Invalid -n argument" 
  network="$3"
fi
```
Also please document the arg in `usage`